backend/app.py

```diff
@@ -25,7 +27,8 @@
 def create_app():
     """Application factory function"""
     app = Flask(__name__)
-    app.config.from_object("backend.config.DevelopmentConfig")
+    env = os.environ["FLASK_ENV"].title()
```

Comment:
this variable does not exist in aws lambda context yet, we have to add that first to not break the page.